src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do list</title>
+</head>
+<body>
+    <header>
+
+    </header>
+
+    <main>
+        <h1>To do list</h1>
+        <div class="container">
+            <div class="container__header">Today's to Do <i id="refresh" class="fa-solid fa-arrows-rotate"></i></div>
+            <div class="container__header"><input id="addTask" type="text" placeholder="Add to your list..." required><i id="enter" class="fa-solid fa-arrow-right-to-bracket"></i></div>
+
+            <ul id="itemsContainer" class="items-container">
+                
+            </ul>
+            <div class="container__footer">
+                <button id="buttonDelete" class="delete-button" type="button">Clear all done</button>              
+            </div>
+        </div>
+       
+    </main>
+
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400&family=Raleway:wght@200;300;500;600&display=swap" rel="stylesheet">
```

Comment:
I think you should use the head when linking CSS or fonts ... It's not fatal but it's a practice recommended by MDN ... take a look here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link

Reply:
Ok great! thank you